src/components/app/MenuButtons/MetaInfo.js

```diff
@@ -21,6 +29,54 @@ type Props = {
 export class MenuButtonsMetaInfo extends React.PureComponent<Props> {
   render() {
     const meta = this.props.profile.meta;
+    const overheads: ?(ProfilerOverhead[]) = this.props.profile
+      .profilerOverhead;
```

Comment:
It's not clear to me why it's an array? How many items are in this array? Is that 1 per process?
Please explain this as a comment in https://github.com/firefox-devtools/profiler/blob/107c0446fe7734bdfcd743446f7e1ea63bf9c745/src/types/profile.js#L503